src/Training/Bundle/UserNamingBundle/EventListener/UserProfileViewListener.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+namespace Training\Bundle\UserNamingBundle\EventListener;
+
+use Oro\Bundle\UIBundle\Event\BeforeListRenderEvent;
+use Oro\Bundle\UserBundle\Entity\User;
+use Twig\Error\LoaderError;
+use Twig\Error\RuntimeError;
+use Twig\Error\SyntaxError;
+
+/**
+ * This class listen to BeforeListRenderEvent and some blocks
+ */
+class UserProfileViewListener
+{
+    /**
+     * Adds sub-block to user profile page in admin panel
+     *
+     * @param BeforeListRenderEvent $event
+     *
+     * @throws LoaderError
+     * @throws RuntimeError
+     * @throws SyntaxError
+     *
+     * @return void
```

Comment:
We usually do not describe throws and do not add docblock if it adds nothing to the method signature